meteor/sofie-server/data-access/repositories/cache/cached-rundown-repository.ts

```diff
@@ -35,4 +36,12 @@ export class CachedRundownRepository implements RundownRepository {
 			this.cachedRundowns.set(rundown.id, rundown)
 		}
 	}
+
+	public async deleteRundown(rundownId: string): Promise<void> {
+		const cacheDeletionResult: boolean = this.cachedRundowns.delete(rundownId)
+		if (!cacheDeletionResult) {
+			throw new DeletionFailedException(`Failed to delete rundown from cache, with rundownId: ${rundownId}`)
+		}
+		await this.rundownRepository.deleteRundown(rundownId)
```

Comment:
It seems a bit off that we first delete it from the cache and then delete it from the database. I would have imagined it the other way around. 
Don't really have any good arguments for either or. So if you have a reason for deleting in cache before the database then I'm curious to hear it :)

Reply:
My reasoning was that because, should it succede in deleting in database, but fail with deleting in cache, then a new `get` request would retrieve from the cache, even though it should not contain the requested element.

If it is the other way around, failing the database, but succeding with the cache, the worst that happens is an extra call to the database, instead of getting stale possibly invalid data.

Hope that made sense.